lib/mimoracle/hf_datasets/mimic_iii/_processing.py

```diff
@@ -0,0 +1,93 @@
+import logging
+import re
+import sqlite3
+from pathlib import Path
+from typing import Optional
+
+import pandas as pd
+import typer
+from tqdm import tqdm
+
+ROOT = Path(__file__).parent.parent.parent
```

Comment:
Perhaps you can add a comment to mention what the ROOT is (it's not really clear it's the root of what if you don't know our practices), it could be something like `# root of the project, where the pyproject.toml is` (if I got it right)